sqlalchemy_risingwave/base.py

```diff
@@ -9,47 +9,19 @@
 
 import sqlalchemy.types as sqltypes
 _type_map = {
-    "bool": sqltypes.BOOLEAN,  # introspection returns "BOOL" not boolean
-    "boolean": sqltypes.BOOLEAN,
-    "bigint": sqltypes.BIGINT,
-    "int": sqltypes.INT,
-    "int2": sqltypes.SMALLINT,
-    "int4": sqltypes.INT,
-    "int8": sqltypes.BIGINT,
-    "int32": sqltypes.INT,
-    "int64": sqltypes.BIGINT,
-    "integer": sqltypes.INT,
-    "smallint": sqltypes.INT,
-    "double precision": sqltypes.FLOAT,
-    "float": sqltypes.FLOAT,
-    "float4": sqltypes.FLOAT,
-    "float8": sqltypes.FLOAT,
-    "real": sqltypes.FLOAT,
-    "dec": sqltypes.DECIMAL,
-    "decimal": sqltypes.DECIMAL,
-    "numeric": sqltypes.DECIMAL,
-    "date": sqltypes.DATE,
-    "time": sqltypes.Time,
-    "time without time zone": sqltypes.Time,
-    "timestamp": sqltypes.TIMESTAMP,
-    "timestamptz": sqltypes.TIMESTAMP,
-    "timestamp with time zone": sqltypes.TIMESTAMP,
-    "timestamp without time zone": sqltypes.TIMESTAMP,
-    "interval": sqltypes.Interval,
-    "char": sqltypes.VARCHAR,
-    "char varying": sqltypes.VARCHAR,
-    "character": sqltypes.VARCHAR,
-    "character varying": sqltypes.VARCHAR,
-    "string": sqltypes.VARCHAR,
-    "text": sqltypes.VARCHAR,
-    "varchar": sqltypes.VARCHAR,
-    "blob": sqltypes.BLOB,
-    "bytea": sqltypes.BLOB,
-    "bytes": sqltypes.BLOB,
-    "json": sqltypes.JSON,
-    "jsonb": sqltypes.JSON,
-    "uuid": UUID,
-    "inet": INET,
+    "boolean": sqltypes.BOOLEAN,  # DataType::Boolean
+    "smallint": sqltypes.SMALLINT,  # DataType::Int16
+    "integer": sqltypes.INT,  # DataType::Int32
+    "bigint": sqltypes.BIGINT,  # DataType::Int64
+    "real": sqltypes.FLOAT,  # DataType::Float32
+    "double precision": sqltypes.FLOAT,  # DataType::Float64
+    "numeric": sqltypes.DECIMAL,  # DataType::Decimal
+    "date": sqltypes.DATE,  # DataType::Date
+    "varchar": sqltypes.VARCHAR,  # DataType::Varchar
+    "time without time zone": sqltypes.Time,  # DataType::Time
+    "timestamp without time zone": sqltypes.TIMESTAMP,  # DataType::Timestamp
+    "timestamp with time zone": sqltypes.TIMESTAMP,  # DataType::Timestampz
```

Comment:
So now for timestamp type, the string is `timestamp with time zone` or just `timestamp`?

Reply:
Timestamp is "timestamp without time zone". It's the Display trait's output of struct `DataType`. You can check them in `src/common/src/types/mod.rs`.